scripts/docker_release/build_algod_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+ALGOD_INSTALL_TAR_FILE=$1
+if [[ $ALGOD_INSTALL_TAR_FILE == "" ]]
+then
+   echo "specify location of base install file"
+   exit 1
+fi
+
+echo "building 'Dockerfile_algod_release' with install file $ALGOD_INSTALL_TAR_FILE"
+cp $ALGOD_INSTALL_TAR_FILE ./temp_install.tar.gz
```

Comment:
Any reason we need to copy this?

Reply:
I had issues with using a non relative path.